### PR TITLE
story/22: Dcat-no wizard

### PR DESCRIPTION
**Pull request by AndreasJJ**
Didn't have time to add tests.

There's also considerable improvements that could be added, like the selection of URLs, selection of release state and improved selection of Category type. But that is for another day.

### CODE REVIEW

**Comment by petterroea** [comment]
Is it possible to add _some_ tests?

**Comment by fipa98** [comment]
Yeah. It does not have to be a part of this branch tho. Will include tests in the upcomming issue.

**Comment by MittBrukernavn** [comment]
Can't seem to find any links directing to the wizard - does the user have to manually go to /wizard to see it?

**Comment by AndreasJJ** [comment]
> Can't seem to find any links directing to the wizard - does the user have to manually go to /wizard to see it?

Yeh, idk where to put it unless we wanna make the header even longer. Also not a part of my issue.

**Comment by MittBrukernavn** [comment]
Could maybe add a link to it on the "submit"-page? I understand that might get crowded as well, but there's a lot more space left there than on the header

**Comment by AndreasJJ** [comment]
at http://localhost:3000/sendData ?

**Comment by MittBrukernavn** [comment]
Yeah, a link to it from there, similar to the "Are none of these types appropriate? Create a new one". Probably not optimal, but fairly clean and an easy fix at the very least

File: frontend/src/pages/DCATWizard/Submission.jsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+        redirect: 'follow',
+        referrerPolicy: 'no-referrer',
+        body: JSON.stringify({
+          name: mimetype.split('/')[1],
+          description: '[No Description]',
+          mimeType: mimetype,
+          documentationUrl: 'http://google.com/fuck+this',
```
Comment (MittBrukernavn) [inline frontend/src/pages/DCATWizard/Submission.jsx:61]: Probably better off left blank?
```suggestion
          documentationUrl: '',
```

File: frontend/src/lib/dcat-ld.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  const distributions = {};
+
+  // const wantedLang = 'no';
+
+  // Find distributions
+  for (const element of ld['@graph']) {
+    if (element['@type'] === 'dcat:Distribution') {
+      const desc = typeof element['dct:description'] === 'undefined'
+        ? '[Ingen beskrivelse]'
```
Comment (MittBrukernavn) [inline frontend/src/lib/dcat-ld.js:40]: Some inconsistency in choice of language (`[No Description]` vs `[Ingen beskrivelse]`)

File: frontend/src/pages/DCATWizard/Submission.jsx (hunk 1 above)
Comment (AndreasJJ) [reply frontend/src/pages/DCATWizard/Submission.jsx:61]: Can it be left blank? @petterroea

File: frontend/src/lib/dcat-ld.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  return dcatExports;
+}
+
+export default parseJsonLd;
+
+// console.log("Extracted data:");
+// console.log(JSON.stringify(dcatExports, null, 4));
+
+// console.log("Extracted distributions:")
+// console.log(JSON.stringify(distributions, null, 4))
```
Comment (MittBrukernavn) [inline frontend/src/lib/dcat-ld.js:98]: These lines should probably be deleted at this point

File: frontend/src/pages/DCATWizard/Submission.jsx (hunk 1 above)
Comment (petterroea) [reply frontend/src/pages/DCATWizard/Submission.jsx:61]: It really shouldnt, but i guess we can change that.

Comment (MittBrukernavn) [reply frontend/src/pages/DCATWizard/Submission.jsx:61]: Or just change it to google.com, just don't think we should push needless profanity to our final product

**Comment by petterroea** [comment]
@MittBrukernavn aight what about now

**Pull request merged**